Android_Assignment_Activity/app/src/main/res/drawable/button_selection.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item android:drawable="@drawable/button_drawn"
```

Comment:
For pressed state you can have drawable and use the default drawable for all remaining states. No need to have separate tag for android:state_pressed="false"